### DISCUSSION TITLE
download does not download

### DISCUSSION THREAD

**Discussion by thedinglebingle** [community/community#75951] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Literally. Everytime i want to download a single file from "releases", it just got damn does not download and freezes the 
Look at the top left. (i clicked the .dll file to download it)
![image](https://github.com/community/community/assets/147722678/9cf9a69e-a84d-4f0a-a77e-a2544734bb86)
How to download?!

**Comment by thedinglebingle** [comment]
wait, how to attach files

**Comment by peterroth** [comment]
As long as the files are uploaded and available, GitHub has not much to do with your problem.
Check in another browser, maybe that blocks the download. If it doesn't help, check if you can download the files on a phone or tablet on the same network, so you can narrow down the problem to your PC/laptop. If on the same network no device can download, contact your internet provider.

**Reply by thedinglebingle** [reply]
the issue seems to randomly appear on any device i have, sometimes it's my phone and somethimes it's my pc. it's weird.
